src/d3a/models/area/stats.py

```diff
@@ -50,6 +52,21 @@ def update_area_market_stats(self):
                 if "bills" in self.aggregated_stats else None
             self.rate_stats_market[self.current_market.time_slot] = \
                 self.min_max_avg_median_rate_current_market()
+            # TODO: This accumulation of trade data could potentially also used for the
```

Comment:
No need to do anything, but I think that there is a need to somehow manage the data that we accumulate over time, instead of just saving the data in each individual class that is part of the endpoint buffer. I mean that we could reuse this class in the future, instead of saving copies of our results in the MarketEnergyBills, DeviceStatistics ... and other classes in the endpoint_buffer.

Reply:
This is actually what i wanted to say in the comment. :-) Shall I rephrase it?

Reply:
Ah great then! It is fine as it is, no need to rephrase it.